### ISSUE TITLE
Numbers and Name Disappear

### ISSUE THREAD

**Issue by koopa86** [AlexModGuy/RetroDamageIndicators#10]
Hello, the mod is excellent, but I am having an issue. The mod starts out working well, but within 3-5 minutes it stops displaying the name of the unit and the HP number. The bar green HP bar still works.

**Comment by thealflandon** [comment]
Turning on the HUD Name and Health Outline to ON fixed this for me. Idk why it happens in the first place.
![image_2024-08-04_091448109](https://github.com/user-attachments/assets/b66fde2a-8c74-455a-8e0d-f67e48ed5821)

**Comment by SlayDigger** [comment]
Had the same issue and this temporary fix worked. Thanks!

**Comment by thealflandon** [comment]
Absolutely! Like I said this also happened to me like mid-way through my playthrough and was tinkering around and found the fix. Must me a render or ingame shader bug, as I don't play with installed shaders.
